setup/containers/docker-compose.yml

```diff
@@ -88,7 +88,7 @@ services:
     image: mcr.microsoft.com/azurestorage-azurite:2.6.7
     container_name: azurite
     volumes:
-      - azurite1:/opt/azurite/data
+      - azurite1:/opt/azurite/folder
```

Comment:
Why has this changed?

Reply:
QueryStore table storage was empty every time I started the container. Maybe the newer image changed the directory used.